packages/server/package.json

```diff
@@ -39,7 +39,7 @@
     "ansi_up": "5.0.0",
     "ast-types": "0.13.3",
     "base64url": "^3.0.1",
-    "better-sqlite3": "9.2.2",
+    "better-sqlite3": "10.0.0",
```

Comment:
@ryanthemanuel Do you remember the context as to why this was necessary?

Reply:
I do not unfortunately. We can try reverting?

Reply:
@AtofStryker do you happen to remember either? I know you were working in here too at times.

Reply:
I was just wondering - I don't think have confirmation that it's breaking anything - it was just a difference between 29 and 28

Reply:
This is needed due to this failure when it is left on 9.2.2: https://cloud.nx.app/runs/n23a1Mbn3x